CHANGELOG.md

```diff
@@ -15,6 +15,10 @@ and this project adheres to
 - @cosmjs/stargate: Let `calculateFee` handle fee amounts that exceed the safe
   integer range.
 
+### Fixed
+
+- @cosmjs/tendermint-rpc: Fix block results validator update decoder. ([#1151])
```

Comment:
The [#1151] notation is a markdown reference for which you need to define a link target like this:

```
[#1151]: https://github.com/cosmos/cosmjs/pull/1151
```